docs/Splashkit/Applications/Arcade Machines/Arcade Machine Setup/Setup Arcade Machine.md

```diff
@@ -126,20 +125,28 @@ and one to wpa_supplicant:
      curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel STS
      ```
 
-  2. Add the following to the end of the file `~/.bashrc` with these commands
+  1. Add the following to the end of the file `~/.bashrc` with these commands
 
      ```shell
      echo 'export DOTNET_ROOT=$HOME/.dotnet' >> ~/.bashrc
      echo 'export PATH=$PATH:$HOME/.dotnet' >> ~/.bashrc
+     echo 'export DOTNET_ROOT=$HOME/.dotnet' >> ~/.bash.bash_profile
```

Comment:
Should this be
```suggestion
     echo 'export DOTNET_ROOT=$HOME/.dotnet' >> ~/.bash_profile
```

Reply:
@Wills2022  Wondering about this too, just want to double check.